lib/pages/items_table.dart

```diff
@@ -161,13 +166,15 @@ class ItemsTable extends StatelessWidget {
     ]);
   }
 
-  http.Response getItems() {
-    var items;
-    http.get('http://127.0.0.1:8000/api/items').then(
-      (value) {
-        items = value;
-      },
-    );
-    if (items != null) return items;
+  getItems() async {
+    final items = await http.get('http://127.0.0.1:8000/api/items');
```

Comment:
I know this does not work but create a `config/development.dart` file where you can place `port` and `address`. Access it from there.